bootstrap_sqlserver.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+# The output of all these installation steps is noisy. With this utility
+# the progress report is nice and concise.
+function install {
+    echo installing $1
+    shift
+    apt-get -y install "$@" >/dev/null 2>&1
+}
+
+# TinyTDS
+install libc6-dev libc6-dev
+wget http://www.freetds.org/files/stable/freetds-1.1.32.tar.gz
+tar -xzf freetds-1.1.32.tar.gz
+cd freetds-1.1.32
+./configure --prefix=/usr/local --with-tdsver=7.3
+make
+make install
+
+# Install SQL Server
+wget -qO- https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -
+add-apt-repository "$(wget -qO- https://packages.microsoft.com/config/ubuntu/18.04/mssql-server-2017.list)"
```

Comment:
Done

Reply:
I think you missed this one.

Instead of 

```
add-apt-repository "$(wget -qO- https://packages.microsoft.com/config/ubuntu/18.04/mssql-server-2017.list)"
```

could you use a separate repository file as you did for `prod.list`?

```
curl https://packages.microsoft.com/config/ubuntu/18.04/mssql-server-2017.list | sudo tee /etc/apt/sources.list.d/microsoft.mssql-server-2017.list
```

Reply:
Done